### PR TITLE
Enable service discovery to query _lifecycle endorsers

### PR DESCRIPTION
**Pull request by wlahti**
#### Type of change

- New feature

#### Description

Enable service discovery to query the necessary endorsers for committing a _lifecycle chaincode definition.

#### Related issues

https://jira.hyperledger.org/browse/FAB-17310

### CODE REVIEW

**Review by yacovm — APPROVED** [review]

**Review by denali49 — COMMENTED** [review]
Not sure doc maintainer approval is needed here, perhaps code maintainer approval only.

**Pull request merged**
